app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.java

```diff
@@ -105,6 +105,12 @@
                 // disabled polling of notifications, which is were the passive polling takes place.
                 NotificationPollBroadcastReceiver.pollEditorTaskCounts(WikipediaApp.getInstance());
             }
+
+            if (WikipediaApp.getInstance().isSurveyLive() && isSuggestedEdits()) {
+                Prefs.setSuggestedEditsCountForSurvey(Prefs.getSuggestedEditsCountForSurvey() + 1);
+                Prefs.updatePrefsToShowSuggestedEditsSurvey();
```

Comment:
Maybe we can leave the logic of updating the Prefs here instead of having them inside the method.